spec/models/institution_spec.rb

```diff
@@ -207,7 +207,7 @@
 
       it 'should filter on field not existing' do
         expect(described_class.filter('institution', 'false').to_sql)
-          .to include("WHERE (\"institutions\".\"institution\" != 't')")
+          .to include("WHERE \"institutions\".\"institution\" != 't'")
```

Comment:
Did this break because of the upgrade? Its not a worrisome change -- just a surprise.

Reply:
It did, it appears the sql generation changed